test/client.test.ts

```diff
@@ -729,7 +729,7 @@ if (!isDeno) {
       new Duration(0, 0, 0, 0, 0, 0, 0, -1),
       new Duration(0, 0, 0, 0, 0, 0, 0, 1),
       new Duration(0, 0, 0, 0, 0, 0, 0, -1),
-      new Duration(0, 0, 0, 0, 0, 0, 0, -752043.296),
+      new Duration(0, 0, 0, 0, 0, 0, 0, -752043),
```

Comment:
Duration no longer supports micros?

Reply:
It does, it just no longer supports non-integer args to the Duration constructor. https://tc39.es/proposal-temporal/#sec-temporal-duration-objects